lambdas/src/common/security/jwt-verifier.ts

```diff
@@ -27,7 +27,7 @@ export class JwtVerifier {
             const signingPublicJwkBase64 = this.jwtVerifierConfig.publicSigningJwk;
             const signingAlgorithm = this.jwtVerifierConfig.jwtSigningAlgorithm;
             const signingPublicJwk = JSON.parse(Buffer.from(signingPublicJwkBase64, "base64").toString("utf8"));
-            const publicKey = await importJWK(signingPublicJwk, signingPublicJwk.alg);
+            const publicKey = await importJWK(signingPublicJwk, signingPublicJwk.alg || signingAlgorithm);
```

Comment:
Maybe `signingPublicJwk?.alg`

Reply:
I think it's coming through as an undefined property anyway, but I can totally defensively code :+1: